src/components/table/__spec__.js

```diff
@@ -902,7 +902,7 @@ describe('Table', () => {
 
       it('returns the emptyRow if no children and has received data', () => {
         let data = Immutable.fromJS([]),
```

Comment:
If a variable is not intended to be reassigned it should be declared with `const`